### DISCUSSION TITLE
Please remove f2a！！

### DISCUSSION THREAD

**Discussion by calidion** [community/community#69315] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My projects are of low severity.
They don't need Two-factor authentication
 to be protected!

**Comment by rnldvrgl** [comment]
1, Access Your Account Settings:
        Click on your profile picture in the top-right corner of the GitHub interface, and then click on "Settings."
2. Security Settings:
        In the left sidebar, click on "Security" to access your security settings.
3. Two-Factor Authentication:
        Scroll down to the "Two-factor authentication" section.
4.Remove 2FA:
        Click the "Disable" button next to "Two-factor authentication." GitHub will prompt you to confirm your password.
5.Confirm Password:
        Enter your GitHub password to confirm that you want to disable 2FA.
6.Disable 2FA:
        Once your password is confirmed, GitHub will disable 2FA for your account.

Please note that after disabling 2FA, your account will be less secure

**Reply by calidion** [reply]
![Screenshot from 2023-10-05 23-22-26](https://github.com/community/community/assets/131776/dd033e36-5b60-4a3a-9158-8d9d13c74c00)

don't have that option.

**Reply by rnldvrgl** [reply]
I think that's because your a part of an organizations, according to this documentation. This might be the reason why you don't have that option.
https://docs.github.com/en/organizations/keeping-your-organization-secure/managing-two-factor-authentication-for-your-organization/requiring-two-factor-authentication-in-your-organization

**Reply by calidion** [reply]
That is fucking ridiculous. I am gonna lose my repositories some day for certain.

**Reply by rnldvrgl** [reply]
If there's no opt-out, I too will just move my repositories to gitlab.

Here's how to migrate things: https://docs.gitlab.com/ee/user/project/import/github.html

**Reply by calidion** [reply]
I have this plan but I just wondering if I can

**Reply by ghost** [reply]
This is false information. There is no disable option even when not a member of any organization.

**Reply by rnldvrgl** [reply]
What do you mean? I have the disable option, this is my profile settings
![image](https://github.com/community/community/assets/77273407/b9c7e354-6291-4e8b-8aed-c8c278396d42)

**Reply by spcharc** [reply]
> Why was my account selected for mandatory 2FA?
> 
> You have taken some action on GitHub that showed you were a contributor.

Quoted from https://github.com/orgs/community/discussions/68240

Basically means, if (you are not in an organization && you did not contribute any code), you can opt out 2fa

**Reply by calidion** [reply]
2FA will surely rob my account in the end!
I will surely lose the second authentication method.
Because no one can remember so many 2FA for every web sites.
The most stupid way for security enhancement.

**Reply by calidion** [reply]
It is like a dictator to force you to accept his order forbidding any question by with the assumption that he is doing good things to you.

**Reply by DzymFardreamer** [reply]
I am in the same situation and I have been enforced to use f2a. I dont want to. I am not sure why I being tagged as a contributtor but if would be nice to let me know what I need to delete on my account to remove the f2a need.

**Comment by nubelaeye** [comment]
❤️

**Comment by calidion** [comment]
remove 2fa!!!!

**Comment by Eikosa** [comment]
I'm having this problem too, please switch off 2fa

**Comment by rminstrel** [comment]
same here! Please remove 2fa!!! I really don't need it

**Comment by Seffi656** [comment]
While your projects may be of low severity, enabling Two-Factor Authentication (2FA) adds an extra layer of security to your account. Even non-critical projects can be targeted by attackers, potentially compromising your credentials and affecting other repositories or services. 2FA helps protect your account from unauthorized access, even if your password gets leaked or stolen.

**Reply by calidion** [reply]
should be optional not forced.  You are force me to lose my account!

**Reply by rminstrel** [reply]
I did not consent to this, rather I am forced to enable 2FA since without it I will lose my account. This is tyranny! I cannot use services linked to GitHub because of 2FA!

**Reply by Seffi656** [reply]
> should be optional not forced. You are force me to lose my account!

I am not forcing you, but don't do this—just remove 2FA.

**Reply by rminstrel** [reply]
> > should be optional not forced. You are force me to lose my account!
> 
> I am not forcing you, but don't do this—just remove 2FA.

Well, I can't. The option is totally missing!

**Reply by rminstrel** [reply]
Somebody really just put an XSS attack on this thread 💀

**Reply by calidion** [reply]
> > should be optional not forced. You are force me to lose my account!
> 
> I am not forcing you, but don't do this—just remove 2FA.

 I can't. It is mandatory for me. becuase I create a lot of open source projects.

**Reply by rminstrel** [reply]
> > I am not forcing you, but don't do this—just remove 2FA.
> 
>  I can't. It is mandatory for me. becuase I create a lot of open source projects.

Same here with me. I submitted a feature request in Jeffser/Alpaca and reported a DDoS attack to nix-community/home-manager and now they're keeping me as well in the shackles of 2FA. Doesn't matter anyways, because I'm thinking about moving my repos to GitLab.

**Comment by comunelonigo** [comment]
the only way to win is not to play

dump github

**Comment by Eloren1** [comment]
Fork anything: 1 click

Delete that fork: multiple clicks, typing name confirmation, then 2fa, are you kidding
